content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -1,215 +1,157 @@
+Given the following `User` table in the database:
 
-![user-table](https://i.imgur.com/EPslYJy.png)
+![user-table](https://imgur.com/f5SFwjC.png)
 
-The equivalent SQL in PostgreSQL dialect:
+This is what the corresponding entity class would look lie:
 
-```sql
-CREATE TABLE "public"."User" (
-  id SERIAL PRIMARY KEY NOT NULL,
-  first_name VARCHAR(255),
-  last_name VARCHAR(255),
-  email VARCHAR(255) UNIQUE NOT NULL,
-  email_confirmed BOOLEAN NOT NULL DEFAULT FALSE,
-  birth_date DATE NOT NULL,
-);
+```ts
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm'
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number
+
+  @Column({ name: 'first_name' })
+  firstName: string
+
+  @Column({ name: 'last_name' })
+  lastName: string
+
+  @Column({ unique: true })
+  email: string
+}
 ```
 
-Here's what its corresponding model class will look like with Sequelize.
-You define the model's fields, map them to the table's fields, and define a class and instance method:
+### Migration workflows
+
+A central part of developing applications that make use of a database is changing the database schema to accommodate new features and to better fit the problem you're solving. In this section, we'll discuss what migrations are and how they affect the workflow.
 
-```js
-import { Model } from 'sequelize'
+Because the ORM sits between the developer and the database, most ORMs provide a **migration tool** to assist with the creation and modification of the database schema.
 
-class User extends Model {
-  // Class method that can be called directly on the model class
-  static async isEmailConfirmed(email) {
-    const count = await this.count({ where: { email, emailConfirmed: true } })
-    return count === 0
+A migration is a set of steps to take the database schema from one state to another. The first migration usually creates tables and indices. Subsequent migrations may alter tables, introduce new indices, or create new tables. Depending on the migration tool, the migration may be in the form of SQL statements or programmatic code which will get converted to SQL statements (as with [ActiveRecord](https://guides.rubyonrails.org/active_record_migrations.html) and [SQLAlchemy](https://alembic.sqlalchemy.org/en/latest/tutorial.html#create-a-migration-script)).
+
+Because databases usually contain data, migrations assist you with breaking down schema changes into smaller units which helps avoid inadvertent data loss.
+
+Assuming you were starting a project from scratch, this is what a full workflow would look like: you create a migration that will create the `User` table in the database schema and define the `User` entity class as in the example above.
+
+Then, as the project progresses and you decide you want to add a new `salutation` column to the `User` table, you would create another migration which would alter the table and add the `salutation` column.
+
+Let's take a look at a TypeORM migration, will be used.
```

Comment:
line break after this line is not reflected in rendered markdown. by design or accidental?